SplunkRumCrashReporting/SplunkRumCrashReporting/CrashReporting.swift

```diff
@@ -83,7 +81,9 @@ func loadPendingCrashReport(_ data: Data!) throws {
     if report.hasExceptionInfo {
         exceptionType = report.exceptionInfo.exceptionName
     }
-
+    guard report.customData != nil  else {
```

Comment:
Nitpick, but `if report.customData === nil` ?

Might also add a debug log here that it's unable to create a crash report due to missing custom data

Reply:
guard report.customData != nil  else {
        SplunkRum.debugLog("Unable to create a crash report due to missing custom data")
            return
    }

Reply:
we are storing SessionId in customData.without SessionId span will not be created.

Reply:
No, if the old session ID can't be found/read we should still send the span with as much data as we have.

Reply:
can we create a new session ID? if the old one doesn't exist.

Reply:
I think creating a new session ID might be confusing, e.g. if you have a normal, working RUM session and then you see an attached crash report for it. So perhaps it's better to omit the session ID if it can't be loaded and send everything else?

Reply:
crash.rumSessionId attribute omitted.